test_metannotate_end_to_end.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+"""
+Created by: Metannotate Team (2017)
+
+Description: A shell script that runs a sample test of Metannotate and compares the generated
+             output files to verify proper end-to-end functionality of the program.
+
+Requirements: - The following packages:
+                    - build-essential
+                    - python-dev
+                    - curl
+                    - file
+                    - git
+                    - python-setuptools
+                    - ruby
+              - Base installation (sudo -H bash base_installation.sh) of Metannotate
+
+"""
+
+python run_metannotate.py --orf_files=data/MetagenomeTest.fa --hmm_files=data/hmms/RPOB.HMM --reference_database=data/ReferenceTest.fa --output_dir=test_output --tmp_dir=test_tmp --run_mode=both
+
+echo "Verifying outputs..."
+
+cd testing/test_constants
+
+# Get reference hash file checksums
+
+FIRST_HASH=$(md5sum rpoB_0_msa_0.fa | awk '{ print $1 }')
```

Comment:
@KyleBehiels You don't need awk. That is redundant. ```awk '{ print $1 }'``` literally prints ```$1``` which is the piped input.

```shell
echo "yolo"
yolo
echo "yolo" | awk '{ print $1 }'
yolo
```